app/helpers/defence_requests_helper.rb

```diff
@@ -87,4 +87,12 @@ def initial_date_type(initial_date, today, tomorrow)
     end
   end
 
+  def appropriate_adult_reason(defence_request)
+    if defence_request.appropriate_adult?
```

Comment:
You don't need to do this check again as the `boolean_with_explanation` helper does it already.